src/components/AnonymousChat/child/ChatReplyPreview.tsx

```diff
@@ -121,6 +140,25 @@ const styles = StyleSheet.create({
     marginLeft: 5,
     padding: 5
   },
+  mediaContainer: {
+    width: 44,
+    height: 44,
+    position: 'absolute',
+    right: 0,
+    top: 0
+  },
+  mediaImage: {
+    width: '100%',
+    height: '100%'
+  },
+  mediaOverlay: {
+    position: 'absolute',
+    backgroundColor: 'rgba(0,0,0,0.5)',
+    width: '100%',
+    height: '100%',
+    justifyContent: 'center',
+    alignItems: 'center'
+  },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New styles have been added for displaying media attachments in chat messages. These styles are used to position and size the media preview and overlay the play icon on video attachments. Ensure these styles work as expected across different screen sizes and resolutions.
```diff
+   mediaContainer: {
+     width: 44,
+     height: 44,
+     position: 'absolute',
+     right: 0,
+     top: 0
+   },
+   mediaImage: {
+     width: '100%',
+     height: '100%'
+   },
+   mediaOverlay: {
+     position: 'absolute',
+     backgroundColor: 'rgba(0,0,0,0.5)',
+     width: '100%',
+     height: '100%',
+     justifyContent: 'center',
+     alignItems: 'center'
+   },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->